pages/common/searchsploit.md

```diff
@@ -12,18 +12,26 @@
 
 `searchsploit sudo 1.8.27`
 
-- Show the exploit-db link to the found resources:
+- Show the exploit-db link (--[w]ww) to the found resources:
```

Comment:
I didn't find a better way to mention the short option

Reply:
```suggestion
- Show the exploit-db link to the found resources:
```
I don't think the short option is required here. If you really need it, then you can go with this:

```suggestion
- Show the exploit-db link to the found resources ([w]):
```